docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/10-advanced-with-helm.md

```diff
@@ -1,9 +1,38 @@
 ---
 sidebar_label: With Helm
+description: Learn how to perform an advanced installation of Kargo using Helm
 ---
 
 # Advanced Installation with Helm
 
-:::warning
-Placeholder
-:::
+For a customized and secure Kargo setup, follow these advanced installation steps.
+
+## Installation Steps
+
+1. Extract the default values from the Helm chart and save it to a convenient
+location. In the example below, we save it to `~/kargo-values.yaml`
+
+    ```shell
+    helm inspect values \
+    oci://ghcr.io/akuity/kargo-charts/kargo > ~/kargo-values.yaml
+    ```
+
+1. Edit and save the values.
+
+    :::info
+    You will find this configuration file contains helpful comments for every
+    option, so specific options are not covered in detail here.
+    :::
+
+1. Proceed with installation, using your modified values:
+
+    ```shell
+    helm install kargo \
+    oci://ghcr.io/akuity/kargo-charts/kargo \
+    --namespace kargo \
+    --create-namespace \
+    --values ~/kargo-values.yaml \
+    --wait
+    ```
+
+For more information on the available configuration options, refer to the [Kargo Helm Chart README](https://github.com/akuity/kargo/tree/main/charts/kargo).
```

Comment:
I think this could be moved up into the info box, as this detail is most relevant right after instructing readers to "edit and save the values" and after having also mentioned the that values files is annotated with helpful comments.

Reply:
A link to operator-guide/security/secure-configuration with a mention that it covers important security related configuration in-depth could also be quite helpful.